### PR TITLE
Including declarations in your npm package

### PR DESCRIPTION
**Pull request by cawa-93**
Declaration files are already created. I only added certain changes to the `package.json`, following [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html).

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -103,9 +103,9 @@
     "@types/node" "*"
 
 "@types/minimatch@*":
-  version "3.0.3"
-  resolved "https://registry.yarnpkg.com/@types/minimatch/-/minimatch-3.0.3.tgz#3dca0e3f33b200fc7d1139c0cd96c1268cadfd9d"
-  integrity sha512-tHq6qdbT9U1IRSGf14CL0pUlULksvY9OZ+5eEgl1N7t+OA3tGvNpxJCzuKQlsNgCVwbAs670L1vcVQi8j9HjnA==
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/@types/minimatch/-/minimatch-3.0.4.tgz#f0ec25dbf2f0e4b18647313ac031134ca5b24b21"
+  integrity sha512-1z8k4wzFnNjVK/tlxvrWuK5WMt6mydWWP7+zvH5eFep4oj+UkrfiJTRtjCeBXNpwaA/FYqqtb4/QS4ianFpIRA==
```
Comment (cawa-93) [inline yarn.lock:108]: These changes have been added by accident. If it's critically, I can cancel them.

File: package.json [hunk 2]
```diff
@@ -51,6 +50,8 @@
     "typescript": "^4.1.4"
   },
   "dependencies": {
+    "@types/rimraf": "^3.0.0",
+    "@types/semver": "^7.3.4",
```
Comment (MarshallOfSound) [inline package.json:54]: Moving these shouldn't be required, the types of these packages are not re-exported as part of our defs?

Comment (cawa-93) [reply package.json]: @MarshallOfSound I guess you right

**Comment by NoahAndrews** [comment]
@MarshallOfSound Do you think you could merge this? There's no up-to-date types available at `electron-devtools-installer' or `@types/electron-devtools-installer`

**Comment by MarshallOfSound** [comment]
Will be handled in modernize --> https://github.com/MarshallOfSound/electron-devtools-installer/pull/254

**Pull request closed without merging**